Framework/PythonInterface/plugins/algorithms/Muon/MuonMaxent.py

```diff
@@ -0,0 +1,441 @@
+import numpy as np
+import math
+from MuonMaxentTools.multimaxalpha import MULTIMAX
+from mantid.api import *
+     # PythonAlgorithm, registerAlgorithm, WorkspaceProperty
+from mantid.kernel import *
+from mantid.simpleapi import *
```

Comment:
It would be better to import `import mantid.simpleapi as api` and then qualify each algorithm usage.

Reply:
I agree, this is suppose to be a quick port of a scientist's code as a temporary solution for MaxEnt. I did not want to make the change as it could be time consuming to work out